vulnerabilities/use-of-deprecated-functions.md

```diff
@@ -6,7 +6,6 @@ Here is a *non-exhaustive* list of deprecated functions and alternatives. Many a
 
 | Deprecated              | Alternatives              |
 | :---------------------- | ------------------------: |
-| `suicide(address)`      |	`selfdestruct(address)`   |
```

Comment:
Maybe makes more sense now to actually include selfdestruct as one of the deprecated functions? e.g.:
```
| `suicide(address)`/`selfdestruct(address)` | N/A |
```